include/pokemon.h

```diff
@@ -330,10 +338,69 @@ struct SpeciesInfo /*0x24*/
  /* 0x16 */ u8 eggGroups[2];
  /* 0x18 */ u16 abilities[NUM_ABILITY_SLOTS]; // 3 abilities, no longer u8 because we have over 255 abilities now.
  /* 0x1E */ u8 safariZoneFleeRate;
- /* 0x1F */ u8 bodyColor : 7;
+            // Pokédex data
+ /* 0x1F */ u8 categoryName[13];
+ /* 0x1F */ u8 speciesName[POKEMON_NAME_LENGTH + 1];
+ /* 0x2C */ u16 cryId;
+ /* 0x2E */ u16 natDexNum;
+ /* 0x30 */ u16 height; //in decimeters
+ /* 0x32 */ u16 weight; //in hectograms
+ /* 0x34 */ u16 pokemonScale;
+ /* 0x36 */ u16 pokemonOffset;
+ /* 0x38 */ u16 trainerScale;
+ /* 0x3A */ u16 trainerOffset;
+ /* 0x3C */ const u8 *description;
+ /* 0x40 */ u8 bodyColor : 7;
+            // Graphical Data
             u8 noFlip : 1;
- /* 0x20 */ u16 flags;
- /* 0x22 */ u8 gigantamax:1;
+ /* 0x41 */ u8 frontAnimDelay;
+ /* 0x42 */ u8 frontAnimId;
+ /* 0x43 */ u8 backAnimId;
+ /* 0x44 */ const union AnimCmd *const *frontAnimFrames;
+ /* 0x48 */ const u32 *const frontPic;
+ /* 0x4C */ const u32 *const frontPicFemale;
+ /* 0x50 */ const u32 *const backPic;
+ /* 0x54 */ const u32 *const backPicFemale;
+ /* 0x58 */ const u32 *const palette;
+ /* 0x5C */ const u32 *const paletteFemale;
+ /* 0x60 */ const u32 *const shinyPalette;
+ /* 0x64 */ const u32 *const shinyPaletteFemale;
+ /* 0x68 */ const u8 *const iconSprite;
+ /* 0x6C */ const u8 *const iconSpriteFemale;
+ /* 0x70 */ const u8 *const footprint;
```

Comment:
Blocking: I think these `const`s are unnecessary:
```suggestion
 /* 0x48 */ const u32 *frontPic;
 /* 0x4C */ const u32 *frontPicFemale;
 /* 0x50 */ const u32 *backPic;
 /* 0x54 */ const u32 *backPicFemale;
 /* 0x58 */ const u32 *palette;
 /* 0x5C */ const u32 *paletteFemale;
 /* 0x60 */ const u32 *shinyPalette;
 /* 0x64 */ const u32 *shinyPaletteFemale;
 /* 0x68 */ const u8 *iconSprite;
 /* 0x6C */ const u8 *iconSpriteFemale;
 /* 0x70 */ const u8 *footprint;
```